app/controllers/api/consents_controller.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class Api::ConsentsController < ApplicationController
+  skip_before_action :verify_authenticity_token, raise: false
+  rescue_from ActiveRecord::RecordNotFound, with: :not_found
+
+  def create
+    member = Member.find_by_id_and_email! params[:id], params[:email]
```

Comment:
Why do we need to pass both email and id? is it a GDPR compilance need?

Reply:
@rodrei - @joeconey and I thought it best to require both bits of information as the endpoint is public, and this makes it a tiny bit more secure.